### DISCUSSION TITLE
Nutritional values

### DISCUSSION THREAD

**Discussion by prisma7411** [TandoorRecipes/recipes#3391] [Recipes]
Nutritional values ​​per person could also be displayed in the Tandoor Recipe Manager. Some file for this feature is missing from the database e.g.: protein 9 g, fat 34 g, carbohydrates 55 g, 560 kcal I haven't figured out what it could be. Would be helpful because I would like to enter all recipes with these values.

**Comment by smilerz** [comment]
Nutrition values can already be entered for each recipe - they scale to the serving size as opposed to per person.

**Reply by prisma7411** [reply]
ok, but when importing, for example, the following recipe https://www.chefkoch.de/rezepte/49481017157468/Curryrahm-Nudeln-mit-Hackfleisch.html
The nutritional values ​​per serving are not included. Does this value not exist in the DB?

**Reply by smilerz** [reply]
Nutrition values are not imported.  That site, specifically, does not include nutritional values in the structured data.

**Reply by prisma7411** [reply]
sorry, it worked now, whatever the reason :-(
can be closed
